lib/components/app/responsive-webapp.js

```diff
@@ -348,6 +349,10 @@ class RouterWrapperWithAuth0 extends Component {
               component={PrintLayout}
               path='/print'
             />
+            <Route
+              component={PrintFieldTripLayout}
+              path='/printFieldTrip'
+            />
```

Comment:
At some point we should probably figure out a solution to selectively include routes depending on config stuff. For now an issue is probably OK.

Reply:
Noted in #399.